.github/workflows/auth-at22-aks-rg.yml

```
@@ -63,6 +66,11 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
+      - name: Set up Grafana Token
+        uses: Altinn/altinn-platform/actions/terraform/grafana-token
```

actionlint (1.7.7)

specifying action "Altinn/altinn-platform/actions/terraform/grafana-token" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)